빵세-방세현/3주차/3WeekPractice/src/App.jsx

```diff
@@ -0,0 +1,46 @@
+// App.jsx
+import "./App.css";
+import { createBrowserRouter, RouterProvider } from "react-router-dom";
+
+import HomePage from "./components/home";
+import NotFound from "./components/not-found";
+import Movies from "./components/MoviesPage";
+import RootLayout from "./layout/root-layout";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    // element: <HomePage></HomePage>,
+    element: <RootLayout />,
+    errorElement: <NotFound></NotFound>,
```

Comment:
이렇게 단일로 사용하는 컴포넌트는 하나 안에서 닫아주세요!
```suggestion
    errorElement: <NotFound/>,
```

Reply:
네 알겠습니다!